android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginScreen.kt

```diff
@@ -404,7 +406,10 @@ fun ForgotPasswordDialog(
       )
     },
     text = {
-      Text(text = stringResource(R.string.call_supervisor, "012-3456-789"), fontSize = 16.sp)
```

Comment:
if we are removing this line, should we remove the `call_supervisor` string?

Reply:
The string is still necessary. The line has just been moved into the column and separated the contact number .

Reply:
Where is it referenced?